src/modules/extractors/items.extractor.ts

```diff
@@ -134,10 +134,28 @@ export class ItemsExtractor {
   }
 
   private itemNameMap: Map<string, Item>;
+  private aliasNameMap: Map<string, Item>;
+  private fileNameMap: Map<string, Item>;
+
+  private hasUnterminatedParenthesis(input: string): boolean {
+    let openParenthesisCount = 0;
+    let closeParenthesisCount = 0;
+
+    for (const char of input) {
+      if (char === '(') {
+        openParenthesisCount++;
+      } else if (char === ')') {
+        closeParenthesisCount++;
+      }
+    }
 
+    return closeParenthesisCount < openParenthesisCount;
+  }
```

Comment:
While this works, there are "cleaner" solutions IMO.
I would personally replace it with a regex to test for unmatched parenthesis, something like:
```ts
return !/^[^()]*(?:\([^()]*\)[^()]*)*$/.test(input);
```

But that's a fairly ugly regex.
To keep it code-based, I would augment it to match the parenthesis "depth" to ensure `this )(` doesn't pass either:
```ts
let depth = 0;
for (const char of input){
  if(char === '('){
    depth += 1;
  } else if(char === ')'){
    depth -=1;
    if(depth < 0){
      return true;
    }
  }
  return depth !== 0;
}
```